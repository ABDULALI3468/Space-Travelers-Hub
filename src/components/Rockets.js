import React from 'react';
import { useDispatch } from 'react-redux';
import fetchRockets from '../Redux/rockets/rockets';

const Rockets = () => {
  const dispatch = useDispatch();
  dispatch(fetchRockets);

  return <div>The Rockets are listed here!</div>;
};

export default Rockets;
