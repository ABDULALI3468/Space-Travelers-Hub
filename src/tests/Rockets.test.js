import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Rockets from '../components/Rockets';
import store from '../Redux/ConfigureStore';

it('Check if it renders', () => {
  const output = renderer
    .create(
      <Provider store={store}>
        <Rockets />
      </Provider>,
    )
    .toJSON();
  expect(output).toMatchSnapshot();
});
