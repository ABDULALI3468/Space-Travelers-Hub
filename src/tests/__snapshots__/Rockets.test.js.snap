// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if it renders 1`] = `
<div
  className="rockets-container"
/>
`;
